trulens_eval/trulens_eval/keys.py

```diff
@@ -129,6 +129,8 @@
             r'KAGGLE_KEY',
             r'SLACK_(TOKEN|SIGNING_SECRET)',
             # Covers slack-related keys.
+            r'.*PASSWORD.*',
+            # Cover user credentials for i.e. Snowflake connection session
```

Comment:
Not really related to this change, but can we also add _API_TOKEN to this list to cover the replicate key (and possibly others?)